site/en/workflows.md

```diff
@@ -671,9 +794,16 @@ A line detection algorithm is run on every text region of every PAGE in the
 input file group, and a TextLine element with the resulting polygon
 outline is added to the annotation of the output PAGE.
 
-**Note:** If you use `ocrd-tesserocr-segment-line`, which uses only bounding boxes instead of polygon coordinates,
-then you should post-process with the processors described in [Step 12](#step-12-resegmentation-line-level).
-If you use `ocrd-cis-ocropy-segment`, you can directly go on with [Step 13](#step-13-dewarping-on-line-level).
+**Note:** If you use `ocrd-cis-ocropy-segment`, you can directly go on with [Step 13](#step-13-dewarping-on-line-level).
```

Comment:
this is already explained in the old text and could be left out

Reply:
> this is already explained in the old text and could be left out

What do you mean _in the old text_? AFAICS that note is still valid, because there exists a `level-of-operation=region` mode for `ocrd-cis-ocropy-segment` which needs no post-processing (step 12 would be redundant).

Reply:
> What do you mean in the old text?

I think @EEngl52 was referring to the paragraph in region segmentation that tells users to skip region post-processing and line segmentation when using one of the region-and-line segmenters. But you're right, there's a valid reason why people want to skip step 12 if not required.